shell/platform/darwin/ios/ios_surface_metal_impeller.mm

```diff
@@ -55,6 +55,10 @@
   // Flutter needs to read from the color attachment in cases where there are effects such as
   // backdrop filters. Flutter plugins that create platform views may also read from the layer.
   layer.framebufferOnly = NO;
+  // Note: this setting improves the performance of the "CPU TO Display Latency" metric in
+  // metal system trace. On high frame rate devices, this reduces the time we spend waiting
+  // for drawable availibility.
+  layer.opaque = YES;
```

Comment:
Emm If we set this `opaque=YES`, means our the change to `opaque` is invalid? Since this will be called every frame.

Reply:
I'll remove this

Reply:
Just testing 😄

Reply:
Yeah ^_^